client/components/mma/paymentUpdate/CurrentPaymentDetail.tsx

```diff
@@ -262,6 +263,7 @@ export const CurrentPaymentDetails = (props: ProductDetail) => {
 							`}
 						>
 							<DirectDebitDisplay
+								data-qm-masking="blocklist"
```

Comment:
This is another one on a component rather than a HTML element.

Reply:
Good spot -removed